content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+
+Although not required to run Sensu Go, we recommend installing plugins with [assets](#activate-assets) where possible.
+
+### 3. Translate filters
+
+Ruby eval logic used in Sensu 1.x filters has been replaced with JavaScript expressions in Sensu Go, opening up powerful possibilities to filter events based on occurrences and other event attributes.
+As a result, you'll need to rewrite your Sensu 1.x filters in Sensu Go format.
+
+First, review your 1.x handlers to see which filters are being used. Then, using the [filter reference][57] and [guide to using filters][58], re-write your filters using Sensu Go expressions. Also check out the [blog post][59] for a deep dive into Sensu Go filter capabilities.
+
+{{< highlight shell >}}
+# Sensu 1.x hourly filter
+{
+  "filters": {
+    "recurrences": {
+      "attributes": {
+        "occurrences": "eval: value == 1 || value % 60 == 0"
+      }
+    }
+  }
+}
+
+# Sensu Go hourly filter
+  {
+    "metadata": {
+      "name": "hourly",
+      "namespace": "default"
+    },
+    "action": "allow",
+    "expressions": [
+      "event.check.occurrences == 1 || event.check.occurrences % (3600 / event.check.interval) == 0"
+    ],
+    "runtime_assets": null
+  }
+{{< /highlight >}}
+
+### 4. Translate handlers
+
+All check results are now considered events and are processed by event handlers.
+You can use the built-in [incidents filter][9] to recreate the Sensu Core 1.x behavior in which only check results with a non-zero status are considered events.
+
+**IMPORTANT**: Silencing is now disabled by default in Sensu Go and must be enabled explicitly using the built-in [`not_silenced` filter][15]. Add the `not_silenced` filter to any handlers for which you want to enable Sensu's silencing feature.
+
+Review your Sensu 1.x check configuration for the following attribute, and make the corresponding updates to your Sensu Go configuration.
+
+| 1.x attribute | Manual updates required in Sensu Go config |
+| ------------- | ------------- |
+`filters: occurrences` | The built-in occurrences filter in Sensu Core 1.x is not provided in Sensu Go, but you can replicate its functionality using [this filter definition][10]. <!-- TODO: replace with link to filter asset guide -->
+`type: transport` | Transport handlers are no longer supported by Sensu Go, but you can create similar functionality using a pipe handler that connects to a message bus and injects event data into a queue.
+`filters: check_dependencies` | Sensu Go does not include a built-in check dependencies filter.
+`severities` | Severities are not available in Sensu Go.
+`handle_silenced` | Silencing is now disabled by default in Sensu Go and must be enabled explicitly using the built-in [`not_silenced` filter][15].
+`handle_flapping` | All check results are now considered events and are processed by event handlers.
+

```

Comment:
handler note:  "history" has changed formats 1.x  was a simple array of status values.   Sensu Go is an array of hashes that contain status and timestamp.  This allows for more expressive filtering and handler logic based on timestamps.

Reply:
Fixed